docs/topics/readme/osio-circuit-breaker-README.adoc

```diff
@@ -11,17 +13,19 @@ From your workspace, click _Stage - 1.0.1_ under _Applications_. This will open
 * Click the blue link next to _Rollout to Run_.
 * Return to your workspace and click _Run - 1.0.1_ under _Applications_ to interact with the _Run_ version of this booster.
 
-== Edit your Booster and Promote the Changes
+== Edit your Booster
 * Click _Create_ at the top of the page.
 * Click _Create Workspace_.
 * Click _Open_.
 +
 NOTE: This will create a popup window so you may have to enable popups in your browser.
 
 * Make a change to your booster (e.g. edit `${fileLocation}`).
+* Click the run button (i.e. the triangle in the top next to the debug button) and choose _run_ to build and deploy your booster in a temporary Che environment. Use the _preview_ url at the top of the log window. Observe that you changes appear.
```

Comment:
Probably a typo:
> Observe that you changes appear.

should probably be:

> Observe that your changes appear.

Also, AFAIK, according to the [style guide](stylepedia.net), we should not use the abbreviation ["i.e."](http://stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-E) in docs. We should use an expanded form, such as "that is", instead.

Multiple instances below.

Reply:
Just a nitpick, but since we write all button names with emphasis and capitalization, maybe we could  do the same for the _run_ button and _preview_ URL. Looks more consistent that way:)